ray-operator/controllers/ray/common/pod.go

```diff
@@ -293,12 +293,6 @@ func initLivenessAndReadinessProbe(rayContainer *corev1.Container, rayNodeType r
 
 // BuildPod a pod config
 func BuildPod(podTemplateSpec corev1.PodTemplateSpec, rayNodeType rayv1.RayNodeType, rayStartParams map[string]string, headPort string, enableRayAutoscaler *bool, creator string, fqdnRayIP string, enableServeService bool) (aPod corev1.Pod) {
-	if enableServeService {
-		// TODO (kevin85421): In the current RayService implementation, we only add this label to a Pod after
-		// it passes the health check. The other option is to use the readiness probe to control it. This
-		// logic always add the label to the Pod no matter whether it is ready or not.
-		podTemplateSpec.Labels[utils.RayClusterServingServiceLabelKey] = utils.EnableRayClusterServingServiceTrue
```

Comment:
Discuss offline: Removing the label would result in the worker Pods lacking this label, preventing them from attaching to the K8s service managed by the RayService.

Reply:
Thank you for pointing out! I have correct it in [6de9391](https://github.com/ray-project/kuberay/pull/1856/commits/6de939138d1cabdca80ff78d33eac10eec30acac). Below is the test:
```shell
kubectl apply -f /home/ubuntu/kuberay/ray-operator/config/samples/ray_v1alpha1_rayservice.yaml
kubectl describe rayservice | grep "Service Status"
# Service Status:  Running
kubectl describe svc rayservice-sample-serve-svc
# Endpoints:         10.244.0.6:8000,10.244.0.7:8000
kubectl describe $(kubectl get pods -o=name | grep worker) | grep "ray.io/serve"
# ray.io/serve=true
kubectl describe $(kubectl get pods -o=name | grep head) | grep "ray.io/serve"
# ray.io/serve=true
```